SQF-VM/SQF_commands.c

```diff
@@ -45,6 +45,35 @@ int64_t system_time_ms(void)
 	return (int64_t)(((timebuffer.time * 1000) + timebuffer.millitm));
 #endif
 }
+
+/**
+ * Checks whether the given array is a 3D vector.
+ * If not an error will be thrown
+ */
+char checkVector3(const PVM vm, const PARRAY array)
+{
+	// size check
+	if (array->top != 3)
```

Comment:
should only check if array has enough elements to be valid in that range `if (array->top < 3)`